src/hyperion/parameters/components.py

```diff
@@ -45,19 +45,64 @@ def __modify_schema__(cls, field_schema):
 PARAMETER_VERSION = ParameterVersion.parse("5.0.0")
 
 
-class RotationAxis(str, Enum):
+class RotationAxis(StrEnum):
     OMEGA = "omega"
     PHI = "phi"
     CHI = "chi"
     KAPPA = "kappa"
 
 
-class XyzAxis(str, Enum):
+class XyzAxis(StrEnum):
     X = "sam_x"
     Y = "sam_y"
     Z = "sam_z"
 
 
+class IspybExperimentType(StrEnum):
```

Comment:
Should: Does it make sense to just have the ones we can do here? It's a bit deceptive to have the whole list when I suspect some of them aren't even used at DLS.

Reply:
I just moved this from the ISPyB modules to avoid circular dependencies, it was already like this. Anyway, I think enums should be exhaustive, it shouldn't hurt us that some are rarely if ever used.